scripts/build-docs-ci

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const child_process = require('child_process');
+const {
+    promisify
+} = require("util");
+const {
+    exec
+} = require('child_process');
+const execAsync = promisify(exec);
+
+const [node, path, releaseTag, ...rest] = process.argv;
+
+const buildDocs = () => {
+  fs.mkdirSync(`./dist`, function(err) {
```

Comment:
Indentation is still inconsistent here. Also, considering the error handling is exactly the same everywhere in this file can we create a function and use it? Something like this

```javascript
const exitOnError = err => {
  if (err) {
    console.error(err):
    process.exit(1);
  }
})

fs.mkdirSync(`./dist`, exitOnError);
```

Reply:
Good point!